src/store/index.js

```diff
@@ -1,20 +1,33 @@
 import Vue from 'vue'
 import Vuex from 'vuex'
 import {getMovies, getScreenings} from '../helpers/index';
+import * as authApi from '../helpers/auth';
+import { setAuthHeader,removeAuthHeader } from '@/helpers/client';
+
+const AUTH_HEADER_STORAGE_KEY = 'auth-header';
 
 Vue.use(Vuex)
 
 export default new Vuex.Store({
   state: {
     movies: [],
     screenings: [],
+    authHeader: null,
```

Comment:
In the future I would consider creating a separate `user` module

Reply:
https://vuex.vuejs.org/guide/modules.html
https://vuex.vuejs.org/guide/modules.html#namespacing